docs/guide/cli/migrate.md

```diff
@@ -0,0 +1,397 @@
+# migrate
+
+The `migrate` command helps you automatically migrate from packages to more performant alternatives. It can replace packages in your codebase with faster, smaller, or more modern alternatives.
+
+## Overview
+
+The migrate command uses an **opinionated codemod** approach based on e18e's curated list of performance-focused package replacements. This ensures consistent, well-tested migrations across the ecosystem.
```

Comment:
we can drop this paragraph and move the list from below to be above the overview heading

then just get rid of overview entirely